### DISCUSSION TITLE
Form表单里的validateTrigger取值范围有哪些啊，文档上都找不到

### DISCUSSION THREAD

**Discussion by zjj2022** [ant-design/ant-design#45354] [Q&A]
只看到有onblur和onChange，这还是示例里面的

**Comment by zombieJ** [comment]
`validateTrigger` 就是把事件注入进去，所以你的子组件接受什么事件就可以填什么事件。相反，如果你的子组件不支持 `onChange` 事件，`validateTrigger` 写 `onChange` 也是没有用的。

**Reply by zjj2022** [reply]
有点似懂非懂， form.item里面一项我在rules里面添加了自定义校验，validateTrigger填的onSubmit，好像也是在我提交的时候触发，但是未不知道它的由来，其他地方没有填validateTrigger

**Reply by Wxh16144** [reply]
https://codesandbox.io/s/biao-dan-xiao-yan-shi-ji-he-yi-lai-shou-ji-5w28ny?file=/demo.tsx

我写了一个 demo， 其实就是取决你的子组件逻辑。 `trigger` 表示收集依赖回调，`validateTrigger` 表示校验时机回调。

demo 里面的 `<MyFoo />` 举例，我需要 onSubmit 中触发校验（点击按钮）这样就需要在 `<Form.Item />` 上设置 validateTrigger 为 onSubmit。

另外一个，修改 value 默认是 onChange，但是  `<MyFoo />`  是 `onMyChange` 所以，对应的 `<Form.Item />`  需要设置 `trigger`  为 `onMyChange`

本质上 formitem 包裹的子组件是一个受控组件。可以打开 demo 试试看就知道区别了
